kuma/wiki/tasks.py

```diff
@@ -37,6 +37,12 @@ def render_document(pk, cache_control, base_url):
     return document.rendered_errors
 
 
+@task()
+def clear_render_stale_document_lock():
+    lock = MemcacheLock('render-stale-documents-lock', expires=60 * 60)
+    lock.release()
+
+
 @task(throws=(StaleDocumentsRenderingInProgress,))
 def render_stale_documents(log=None):
     """Simple task wrapper for rendering stale documents"""
```

Comment:
I can't comment on the line b/c it's not close enough to a change, but is `if lock.acquired` the right conditional below? `lock.acquired` sounds like the successful path? Looking at the code it says there's already a lock so I guess it is just the wording that is awkward.